apiary.apib

```diff
@@ -2217,6 +2217,7 @@ Create a quotation.
                                 + Members
                                     + percentage - Values between 0 and 100
                         + product_id: `d905ff57-e866-0f59-9d1e-1fd4538bfae1` (string, optional) - This is purely informational and does not affect the quotation besides adding a reference to the product
+                        + purchase_price (Money, nullable, optional) - the currency must match the account currency
```

Comment:
PP: there's no point in being able to pass null when creating

Reply:
Seems nice to me that (as a client) you don't have to remove the key-value pair if the value happens to be null.